Kajabi/sage-archived
# Update Sage icon font with refresh icon

## Description
Need to add a `refresh` icon for product work on Sell.

### Screenshots
#### sage-icon-refresh
<img width="144" alt="sage-icon-refresh" src="https://user-images.githubusercontent.com/7331038/97219753-ceffaa00-17a0-11eb-9751-2ebc924fdd6a.png">

### Steps for testing
1. Open http://localhost:4000/pages/foundations/icon
1. Ensure that the `sage-icon-refresh` icon exists on the page
